brave/brave-core
# Adds migration for min visits

Resolves https://github.com/brave/brave-browser/issues/3250
Resolves https://github.com/brave/brave-browser/issues/3277

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- create profile on the latest master
- enable rewards
- add some sites to ac table
- close browser
- open publisher db
- change visits for all entries in activity_info to 0
- save db
- switch to this PR branch
- start browser
- visit one site
- open db again
- make sure that all entries in activity_info have 1 visit recorded

## Reviewer Checklist:
